### PR TITLE
Add framework for localization, translations to come later

### PR DESCRIPTION
**Pull request by alogan519**

### CODE REVIEW

File: d2l-rubric-criteria-group.html [hunk 1]
```diff
@@ -98,12 +99,27 @@
 				},
 				href: String,
 
-				token: String
+				token: String,
+
+				resources: {
```
Comment (CodeBaboon) [inline d2l-rubric-criteria-group.html]: These should probably be imported from lang-specific files.

File: d2l-rubric-criteria-group.html [hunk 2]
```diff
@@ -27,14 +28,14 @@
...
-			<table class="d2l-table" summary="This table lists criteria and criteria group names in the first column. The first row lists level names and includes scores if the rubric uses a numeric scoring method">
+			<table class="d2l-table" summary="{{localize('rubric_summary_a11y')}}">
 				<thead>
 					<tr>
 						<th></th>
 						<template is="dom-repeat" items="[[levels]]">
 							<th>
 								<div class="levels-header">[[item.properties.name]]</div>
-								<div hidden="[[!item.properties.points]]">[[item.properties.points]] points</div>
+								<div hidden="[[!item.properties.points]]">{{localize('points', 'number', item.properties.points)}}</div>
```
Comment (jstefaniuk-d2l) [inline d2l-rubric-criteria-group.html]: These can still all be square brackets

Comment (alogan519) [reply d2l-rubric-criteria-group.html]: ok, not sure why all the examples I see use {{}}

File: d2l-rubric-criteria-group.html (hunk 1 above)
Comment (alogan519) [reply d2l-rubric-criteria-group.html]: Ya that sounds good, will do

File: d2l-rubric-criteria-group.html (hunk 2 above)
Comment (jstefaniuk-d2l) [reply d2l-rubric-criteria-group.html]: Bad habit mostly, I think (since `{{}}` is 2-way binding which isn't used in function calls like this)

**Comment by jstefaniuk-d2l** [comment]
I think Dave was suggesting a setup like this: https://github.com/Brightspace/parent-portal-ui/tree/master/src/lang

File: lang/en.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	window.D2L = window.D2L || {};
+	window.D2L.PolymerBehaviors = window.D2L.PolymerBehaviors || {};
+	window.D2L.PolymerBehaviors.Rubric = window.D2L.PolymerBehaviors.Rubric || {};
+	window.D2L.PolymerBehaviors.Rubric.LocalizeBehavior = window.D2L.PolymerBehaviors.Rubric.LocalizeBehavior || {};
+
+	/*
+	* En lang terms
+	* @polymerBehavior D2L.PolymerBehaviors.Rubric.LocalizeBehavior.LangEnBehavior
+	 */
+	/** @polymerBehavior D2L.PolymerBehaviors.Rubric.LocalizeBehavior */
```
Comment (jstefaniuk-d2l) [inline lang/en.html]: just gotta rename this comment

**Pull request merged**